static/coffee/flows/controllers.coffee

```diff
@@ -1056,27 +1056,31 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
     ruleset.rules[0].category[Flow.flow.base_language] = 'All Responses'
 
   formData.timeoutOptions = [
-    {value:1, text:'1 minute'},
-    {value:2, text:'2 minutes'},
-    {value:3, text:'3 minutes'},
-    {value:4, text:'4 minutes'},
-    {value:5, text:'5 minutes'},
-    {value:10, text:'10 minutes'},
-    {value:15, text:'15 minutes'},
-    {value:30, text:'30 minutes'},
-    {value:60, text:'1 hour'},
-    {value:120, text:'2 hours'},
-    {value:180, text:'3 hours'},
-    {value:360, text:'6 hours'},
-    {value:720, text:'12 hours'},
-    {value:1080, text:'18 hours'},
-    {value:1440, text:'1 day'},
-    {value:2880, text:'2 days'},
-    {value:4320, text:'3 days'},
-    {value:10080, text:'1 week'},
+    {value:10, text:'10 seconds'},
```

Comment:
@srugano - GoFlow it's waiting for minutes (https://github.com/greatnonprofits-nfp/goflow/blob/ccl-current-version/legacy/definition.go#L676), so in your implementation the values are wrong, that is, 10 seconds should be ~0.16, 20 seconds should be ~0.33, and so on.